draft-ietf-rats-architecture.md

```diff
@@ -133,35 +133,35 @@ This document uses the following terms.
 
 Appraisal Policy for Evidence:
 
-: A set of rules that informs how a Verifier evaluates the validity of information about an Attester. Compare /security policy/ in {{RFC4949}}
+: A set of rules, reference values and such that is used by a Verifier to evaluate the Evidence from an Attester. Compare /security policy/ in {{RFC4949}}
 
 Appraisal Policy for Attestation Results:
 
 : A set of rules that direct how a Relying Party uses the Attestation Results regarding an Attester generated by the Verifiers. Compare /security policy/ in {{RFC4949}}
 
 Attestation Result:
 
-: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
+: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results.
```

Comment:
```suggestion
: The output generated by a Verifier, typically including information about an Attester, where the Verifier vouches for the validity of the results
```

Reply:
not a sentence (applies multiple times)

Reply:
"The output generated by a Verifier, that typically includes information about the Attester, that is meaningful to a Relying Party and where the Verifier vouches for the validity of the Attestation Results."

Reply:
Tightening up Ned's text further: "The output generated by a Verifier, that is information about the Attester, that is used by a Relying Party and where the Verifier vouches for the Attestation Results."